orttraining/orttraining/models/bert/main.cc

```diff
@@ -593,7 +593,6 @@ void setup_training_params(BertParameters& params) {
       {"input_mask", "input3"},
       {"masked_lm_positions", "masked_lm_positions"},
       {"masked_lm_ids", "masked_lm_ids"},
```

Comment:
Would existing onnx models fail? for example, e2e/perf test pipelines? Perhaps take the original Bert toy model, and run it, if it works, then it's fine.

Reply:
There is a BERT tiny test that passes but I can run more tests, also will push another commit to sync with master because it seems more tests were added that use BERT-tiny e2e training

Reply:
one is good enough :-)

---
In reply to: [455380227](https://github.com/microsoft/onnxruntime/pull/4509#discussion_r455380227) [](ancestors = 455380227)